feat: Add timeStamps in attached text

thepiyush-303 opened the pull request:
# Brief Title
Now text attachments also include timeStamps beside username just like RC
## Acceptance Criteria fulfillment

- [ ] Reverse the order of showing message before attachment, since RC follow attachment before message so that I reverse the order
- [ ] Include timeStams in both simple and nested-attachment
- [ ] Remove @ sign from attachment because RC also doesn't include it.

Fixes #873 

## Video/Screenshots
Before:
![image](https://github.com/user-attachments/assets/c64a0dce-9bca-4feb-bf52-43fe0ee1fb99)
After this PR it will look like this.
![Screenshot from 2025-01-12 23-23-49](https://github.com/user-attachments/assets/a1fb9db6-e659-426b-bd1a-7a4e0daf72e2)

## PR Test Details

**Note**: The PR will be ready for live testing at https://rocketchat.github.io/EmbeddedChat/pulls/pr-<pr_number> after approval. Contributors are requested to replace `<pr_number>` with the actual PR number.

thepiyush-303 commented:
I think adding `jumpToMessage` on clicking `timeStamp` is not possible because we have only access of `attachment_link`.
am i right @Spiral-Memory

Spiral-Memory commented:
In EC, the choice between username or name is customizable: You will have to make it dynamic :

Remove @ sign from attachment because RC also doesn't include it.

thepiyush-303 commented:
@Spiral-Memory 
Done.
![image](https://github.com/user-attachments/assets/fbe0d283-2394-4b13-b9a0-37c134423f54)



any other changes?

thepiyush-303 commented:
(same video as above)

https://github.com/user-attachments/assets/883434c8-60b2-4909-bee9-20076f60ec75




Hey @Spiral-Memory, 
I have added timestamp in the following way:
- if message is from previous month then show date in 'january 28,2025' format.
- if message is from previous week but same month then show the date '01/02/2025' format
- if message is from same week then show 'day, time'
- is message is from same day then show 'time'.

It is inspired by the approach that RC follows, i implement it and found some responsive issues so i also fixed them, check final changes is video.
